packages/@aws-cdk/aws-eks/lib/kubectl-handler/apply/__init__.py

```diff
@@ -24,6 +24,7 @@ def apply_handler(event, context):
     role_arn      = props['RoleArn']
     prune_label   = props.get('PruneLabel', None)
     overwrite     = props.get('Overwrite', False)
+    skip_validation = props.get('SkipValidation', False)
```

Comment:
Lets consolidate all this falsy/truthy shenanigans here, so we can freely use these as booleans later on in the code.

```suggestion
    skip_validation = props.get('SkipValidation', 'false').lower() == 'true'
```

Same for `overwrite ` :)